lib/types.js

```diff
@@ -133,3 +133,32 @@ TDouble.prototype = {
 };
 
 exports.TDouble = TDouble;
+
+function TTrue (value) { this.value = value; }
+TTrue.prototype = {
+    typetag: 'T',
+    decode: function (data) {
+        this.value = true
+        return data;
+    },
+    encode: function (buf, pos) {
+        return pack('>T', buf, pos, [ this.value ]);
```

Comment:
I think you can avoid using `this.value` here and just make it it true. 
`decode` could just true